sources/packages/backend/apps/db-migrations/src/sql/EducationPrograms/Add-col-none-of-entrance-requirements.sql

```diff
@@ -0,0 +1,7 @@
+-- Add none_of_entrance_requirements column to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN none_of_entrance_requirements BOOLEAN NOT NULL DEFAULT FALSE;
+
+COMMENT ON COLUMN sims.education_programs.none_of_entrance_requirements IS 'True when none of the above entrance requirements is selected.';
```

Comment:
`Above` seems to be targeting the UI, can we change it to be more related to the DB?